apps/vm_introspect/tk1/devices.camkes

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2018, Data61
+ * Commonwealth Scientific and Industrial Research Organisation (CSIRO)
+ * ABN 41 687 119 230.
+ *
+ * This software may be distributed and modified according to the terms of
+ * the BSD 2-Clause license. Note that NO WARRANTY is provided.
+ * See "LICENSE_BSD2.txt" for details.
+ *
+ * @TAG(DATA61_BSD)
+ */
+
+#include <configurations/vm.h>
+
+#define VM_INITRD_MAX_SIZE 0x1900000 //25 MB
+#define VM_PADDR_RAM_BASE 0xB0000000
+#define VM_RAM_BASE 0x80000000
+#define VM_RAM_SIZE 0x40000000
+#define VM_RAM_OFFSET 0x30000000
+#define VM_DTB_ADDR 0x82000000
+#define VM_INITRD_ADDR 0x80700000
+
+assembly {
+	composition {}
+	configuration {
+
+        vm0.memdev_paddr = VM_RAM_BASE;
```

Comment:
Does this need to take into account the offset?

Reply:
Hi, can you confirm the `VM_RAM_BASE + VM_RAM_OFFSET` is the start of the guest memory?

Reply:
I may have limited context here, but using the offset seems to add up to the `VM_PADDR_RAM_BASE` a few lines above + the addresses in the untyped mmios below?

Reply:
Makes sense. Thanks!